webextension/js/popup.js

```diff
@@ -750,10 +751,30 @@ Logic.registerPanel(P_CONTAINER_EDIT, {
     this.initializeRadioButtons();
 
     Logic.addEnterHandler(document.querySelector("#edit-container-panel-back-arrow"), () => {
-      this._submitForm();
+      const formValues = new FormData(this._editForm);
+      if (formValues.get("container-id") !== NEW_CONTAINER_ID) {
+        this._submitForm();
+      } else {
+        Logic.showPreviousPanel();
```

Comment:
This doesn't seem to work? When I edit a container and press "Enter" it doesn't do anything? I expected it to do the same as the back arrow: show the previous panel.